bluetooth-rfkill-event/bluetooth_rfkill_event.c

```diff
@@ -573,6 +607,19 @@ void parse_config(GKeyFile *config)
 
     check_config(config);
 
+    str = g_key_file_get_string(config, "General", "patcher", &err);
```

Comment:
Memory leak? looks like return value from g_key_file_get_string() is not released.